docs/neodys/neodys.rst

```diff
@@ -0,0 +1,75 @@
+.. doctest-skip-all
```

Comment:
Now we do test the code examples in the narrative docs. Please remove this line, and use the `.. doctest-remote-data::` directive instead of `.. code-block:: python` for the code blocks. Some tweaking may be required, but IMO it's totally worth it if the result is a tested documentation.